recipes/libpcap/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(libpcap_example)
```

```suggestion
project(test_package)
```

Please leave this generic. Should be the same for all recipes.